### PR TITLE
New tests for worker

### PR DESCRIPTION
**Pull request by Zelldon**
* Move logic of worker into own package.
  * For better encapsulation and testability
* Write new tests for job worker/handler

### CODE REVIEW

File: go-chaos/worker/chaos_worker.go [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+import (
+	"context"
+	"time"
+
+	"github.com/camunda/zeebe/clients/go/v8/pkg/entities"
+	"github.com/camunda/zeebe/clients/go/v8/pkg/worker"
+)
+
+var CommandRunner func([]string, context.Context) error
```
Comment (oleschoenburg) [inline go-chaos/worker/chaos_worker.go:25]: :question: This is a globally shared, mutable variable, right? I'm not sure if it's idiomatic in go but I'd have expected a `ChaosWorker` struct that encapsulates the handler function. Just to get rid of the global var.

**Review by oleschoenburg — APPROVED** [review]
:+1: I like the refactoring, just one minor comment

File: go-chaos/worker/chaos_worker.go (hunk 1 above)
Comment (Zelldon) [reply go-chaos/worker/chaos_worker.go:25]: Yeah I was not 100% sure where to put it then. But I guess I need a wrapper around the handler to insert more inputs.

Comment (Zelldon) [reply go-chaos/worker/chaos_worker.go]: Good point @oleschoenburg I fixed it :)

**Pull request merged**